### PR TITLE
Requested Changes have been made

### PR DESCRIPTION
**Pull request by Hiwot-Chichaybelu**
All changes requested by @rcrichton have been made. next-gen branch is ready to be merged to the master branch.

### CODE REVIEW

**Comment by rcrichton** [comment]
@Hiwot-Chichaybelu I don't see any new commits. Have you pushed your latest changes to the Next_Gen branch on Hiwot-Chichaybelu/openhim-mediator-openinfoman-dhis2-sync

**Comment by rcrichton** [comment]
Also note, you don't need to open a new PR each time there are changes. Just push to the same branch the Pr is based off of and it will update. I've closed off the older PRs to keep this one as the latest.

File: src/server.js [hunk 1]
```diff
@@ -632,7 +631,8 @@ saveConfigToFile(clientMap);
 
     res.set('Content-Type', 'application/json+openhim');
     return res.send({
-      'x-mediator-urn': mediatorConfig.urn,
+      //'x-mediator-urn': config.getMediatorConf().urn,
```
Comment (rcrichton) [inline src/server.js]: Remove commented out code.

Comment (Hiwot-Chichaybelu) [reply src/server.js]: Removed commented code.

**Pull request merged**